docs/site/decorators/Decorators_openapi.md

```diff
@@ -427,52 +427,6 @@ export class SomeController {
 }
 ```
 
-#### anyOf, allOf, oneOf, not
-
-The `x-ts-type` extention is also valid as a value in `allOf`, `anyOf`, `oneOf`,
-and `not` schema keys.
-
-```ts
-@model
-class FooModel extends Model {
-  @property()
-  foo: string;
-}
-
-@model
-class BarModel extends Model {
-  @property()
-  bar: string;
-}
-
-@model
-class BazModel extends Model {
-  @property()
-  baz: string;
-}
-
-class MyController {
-  @get('/some-value', {
-    responses: {
-      '200': {
-        description: 'returns a union of two values',
-        content: {
-          'application/json': {
-            schema: {
-              not: {'x-ts-type': BazModel},
-              allOf: [{'x-ts-type': FooModel}, {'x-ts-type': BarModel}],
-            },
-          },
-        },
-      },
-    },
-  })
-  getSomeValue() {
-    return {foo: 'foo', bar: 'bar'};
-  }
-}
-```
-
```

Comment:
We should revert this. I think our intention is to keep this section for `anyOf, allOf, oneOf, not`.

Reply:
Yes, I see my other PR landed in so this should not be removed now.